spec/dialogHandler.spec.ts

```diff
@@ -45,6 +45,9 @@ describe('dialog handler', () => {
             const certificate = null;
 
             it('should return false when buttonId is 1', async (done) => {
+                BrowserWindow.fromWebContents = jest.fn(() => {
```

Comment:
This one and the below one can be moved to /spec/__mock__/electron.ts

Reply:
Problem is, this would affect the other `fromWebContents` mocks.